modules/bridge/src/main/ts/ephox/bridge/components/dialog/Tree.ts

```diff
@@ -11,12 +11,23 @@ export interface TreeSpec {
   type: 'tree';
   items: TreeItemSpec[];
   onLeafAction?: (id: Id) => void;
+  defaultExpandedIds?: Id[];
+  onExpand?: (
+    expandedKeys: Id[],
+    { expanded, node }: { expanded: boolean; node: Id }
+  ) => void;
 }
 
 export interface Tree {
   type: 'tree';
   items: TreeItem[];
+  defaultExpandedIds: Optional<Id[]>;
```

Comment:
The way I thought about it, is that it is easier for the user of this component to not provide the option instead of having to provide an empty array. What do you think ?

Reply:
I mean internally wouldn't it be easier to have an array since you do a getOr([]) on it anyway. So just default it to empty array in the schema. But for the integrator it's optional.

Reply:
Ah I see now

Reply:
Done!